playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -317,6 +317,21 @@
     - install
     - install:app-requirements
 
+# --no-save is passed as a flag to npm install to avoid saving these dependencie to package.json. Otherwise,
+# running npm install without this flag causes modifications to the package.json and package-lock.json
+# files. In turn, these modified files cause issues with working with the edxapp repository.
+- name: Install extra node dependencies
+  shell: "easy_install --version && npm install --no-save {{ item.name }}{{ '@' + item.version_range if item.version_range is defined else '' }}"
+  with_items: "{{ EDXAPP_EXTRA_NPM_REQUIREMENTS }}"
+  args:
+    chdir: "{{ edxapp_code_dir }}"
+  environment: "{{ edxapp_environment | combine(git_ssh_environment_mixin) }}"
```

Comment:
On the python side, we will have one list that applies to all environments.
- Is that possible for npm as well?
- I'm not clear on what this is or how it works, but we'll need to re-implement this in a Dockerfile for the containerized version, and I'm not sure what we'll have access to regarding environment.

Reply:
1. Based on your pull request, it appears that the way to share this list between environments is to set it in `ansible/vars/edx.yml` instead of `ansible/vars/{env}-edx.yml`. I don't see why it wouldn't be possible for this list.
2. To be honest, I'm not sure what this value is or what it does. I copied it from the earlier npm-related tasks in this playbook. I imagine that containerization will have to handle these variables in all three npm-related tasks. I'm not sure how best to do that.

Reply:
To summarize, let's **not** make this environment specific. If you run into an issue with this, then please bring it up. Thanks.

Reply:
I removed the line. Can you review and let me know if this looks good to you?